it/test/controllers/individual/YouCanSignUpControllerISpec.scala

```diff
@@ -0,0 +1,29 @@
+package controllers.individual
+
+import helpers.ComponentSpecBase
+import play.api.test.Helpers._
+import play.api.libs.ws.{WSClient, WSResponse}
+import play.api.test.WsTestClient
+import scala.concurrent.{ExecutionContext, Future}
+
+import scala.concurrent.ExecutionContext.Implicits.global
+
+class YouCanSignUpControllerISpec extends ComponentSpecBase {
+
+  s"GET ${routes.YouCanSignUpController.show.url}" must {
+    "return OK with the page content" in {
+
+      WsTestClient.withClient { client =>
+        val fullUrl = s"http://localhost:9000${routes.YouCanSignUpController.show.url}"
+
+        val response: Future[WSResponse] = client.url(fullUrl)
+          .withHttpHeaders("Cookie" -> "session-id=fake-session-id")
+          .get()
+
+        response.map { wsResponse =>
+          wsResponse.status mustBe OK
+        }
+      }
+    }
+  }
```

Comment:
```suggestion
  s"GET ${routes.YouCanSignUpController.show.url}" when {
    "the user is authenticated" must {
      "return OK with the page content" in {
        AuthStub.stubAuthSuccess()
    
        val result = youCanSignUp()
    
        result must have(
          httpStatus(OK),
          pageTitle(...)
        )
      }
      "the user is unauthenticated" must {
        "redirect to the login page" in {
          AuthStub.stubUnauthorised()
          
          val result = youCanSignUp()
          
          result must have(
            httpStatus(SEE_OTHER),
            redirectUri(basGatewaySignIn("/you-can-sign-up-now"))
          )
        }
      }
  }
```

Can be rewritten to use some of the features we use in other tests for consistency, also adding in a test to prove the unauthenticated controller will send users to login is a good test to have.